pydax/loaders/_base.py

```diff
@@ -34,3 +34,14 @@ def load(self, path: Union[_typing.PathLike, Dict[str, str]], options: SchemaDic
         :return: The object representing the loaded file.
         """
         pass
+
+    def check_path(self, path: Union[_typing.PathLike, Dict[str, str]]) -> None:
```

Comment:
I would call this function in the abstract `load()` and let overriding methods call `super().load()`.